Assignment/SampleData.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using static System.Net.Mime.MediaTypeNames;
+using System.Net.NetworkInformation;
+using System.Reflection.Metadata;
+using System.Threading.Channels;
+using System.Text;
+
+namespace Assignment;
+
+public class SampleData : ISampleData
+{
+    //File.ReadLines internally handles the resource management and ensures proper disposal of resources once it finishes reading the lines.
+    //It's designed to be used without requiring you to manually dispose of resources.
+    public IEnumerable<string> CsvRows { get; set; } = File.ReadLines("People.csv").Skip(1);
+
+    public IEnumerable<string> GetUniqueSortedListOfStatesGivenCsvRows()
+    {
+        IEnumerable<string> states = CsvRows.Select(CsvRows => CsvRows.Split(",")[6].Trim()).Distinct().OrderBy(state => state);
+        return states;
```

Comment:
```suggestion
        return CsvRows.Select(CsvRows => CsvRows.Split(",")[6].Trim()).Distinct().OrderBy(state => state);
```
nit: simplify

Even simpler, a expression-bodied member